contracts/BC_fusion/BSCGovernor.sol

```diff
@@ -194,7 +267,7 @@ contract BSCGovernor is
         uint256[] memory values,
         bytes[] memory calldatas,
         bytes32 descriptionHash
-    ) internal override(GovernorUpgradeable, GovernorTimelockControlUpgradeable) {
+    ) internal override(GovernorUpgradeable, GovernorTimelockControlUpgradeable) whenNotPaused {
```

Comment:
why whenNotPaused is only add for a view function?

Reply:
it is not a view function